_includes/manuals/1.16-develop/5.9_http_proxy.md

```diff
@@ -0,0 +1,8 @@
+If Foreman is running behind a Firewall, that has HTTP port 80 and HTTPS port 443 blocked a HTTP proxy can be configured to allow requests to external compute resources, such as AWS EC2, Rackspace or Docker as well as external content provider and registries.
```

Comment:
It does not have to be port 80/443, it applies to all communication with external systems, including foreman proxies. I don't date to suggest the sentence, but I think it should mention all external HTTP communication can be proxied and include few examples such as compute resource, foreman proxy, other endpoints coming from plugins e.g. docker registry.